Marlin/src/Marlin.cpp

```diff
@@ -408,6 +408,7 @@ void manage_inactivity(bool ignore_stepper_queue/*=false*/) {
       if (!homeDebounceCount) {
         enqueue_and_echo_commands_P(PSTR("G28"));
         LCD_MESSAGEPGM(MSG_AUTO_HOME);
+		enqueue_and_echo_commands_P(PSTR("M84"));
```

Comment:
Tabs considered evil. See http://marlinfw.org/docs/development/coding_standards.html for guidelines on code formatting.